app/uk/gov/hmrc/transitmovementsauditing/controllers/AuditController.scala

```diff
@@ -57,12 +61,12 @@ class AuditController @Inject() (
 
   def post(auditType: AuditType, uri: Option[ObjectStoreResourceLocation] = None): Action[Source[ByteString, _]] = Action.async(streamFromMemory) {
```

Comment:
@drnaylor-oc    With Auditing I can see for Large message we are passing Object-Store-URI as part of request param instead of header. also looks like we have created new endpoint for large message
``` POST        /audit/:auditType/*uri ``` 
and inside the controller we have created new method **postLarge** and inside that we are calling another existing endpoint method **post** that looks odd to me. Are you happy with this change? please suggest.

Reply:
For the most part, I suggested that to Derek -- it doesn't bother me so much.

Given the work I have to do on object-store access, this _may_ end up being temporary anyway.

Reply:
We tried a couple of alternatives but this worked out to be the most feasible.